components/docs-chef-io/content/habitat/habitat_cli.md

```diff
@@ -1,3275 +1,3 @@
-+++
-title = "hab CLI Reference"
-draft= false
-
-aliases = ["/habitat/habitat-cli/"]
-
-[menu]
-  [menu.habitat]
-    title = "hab CLI Reference"
-    identifier = "habitat/reference/habitat-cli CLI Reference"
-    parent = "habitat/reference"
-    weight = 10
-+++
```

Comment:
We should keep this page metadata. Hugo won't build without those variables. Same with `service_templates.md`. Otherwise it looks good to me.

Reply:
@IanMadd These are the old files that are destined to be overwritten by the generated ones. Are you saying that we need to have the metadata in place in order to build the site for "everyday" work on the site, too?

Reply:
@christophermaier Yep, leaving that metadata out will make Hugo fail if we try to do build previews with Netlify or just preview the docs locally with `make serve`. The other option is to just delete the files entirely, but I think  it's better to have them there and then include the note stating that the content is generated.

Reply:
@IanMadd Gotcha, thanks for catching that!

Addressed in 58b9714